pkg/controller/pxcbackup/controller.go

```diff
@@ -114,6 +113,19 @@ func (r *ReconcilePerconaXtraDBClusterBackup) Reconcile(request reconcile.Reques
 		return reconcile.Result{}, fmt.Errorf("invalid backup cluster: %v", err)
 	}
 
+	changed, err := cluster.CheckNSetDefaults()
+	if err != nil {
+		return reconcile.Result{}, fmt.Errorf("wrong PXC options: %v", err)
+	}
+
+	// update CR if there was changes that may be read by another cr (e.g. pxc-backup)
+	if changed {
+		err = r.client.Update(context.TODO(), cluster)
```

Comment:
I think we don't need Update CR here, just run `cluster.CheckNSetDefaults`
@dAdAbird what do you think?

@almariah could you please rebase on master?

Reply:
rebased!